colin-api/src/colin_api/models/business.py

```diff
@@ -69,27 +64,19 @@ class CorpStateTypes(Enum):
         VOLUNTARY_DISSOLUTION = 'HDV'
         ADMINISTRATIVE_DISSOLUTION = 'HDA'
 
-    # temp converter because legal doesn't have codes only class (legal_type)
-    CORP_TYPE_CONVERSION = {
-        LearBusinessTypes.COOP.value: [
-            TypeCodes.COOP.value
-        ],
-        LearBusinessTypes.BCOMP.value: [
-            TypeCodes.BCOMP.value,
-            TypeCodes.BC_COMP.value,
-            TypeCodes.ULC_COMP.value
-        ],
-        LearBusinessTypes.BC_COMP.value: [
-            TypeCodes.BCOMP.value,
-            TypeCodes.BC_COMP.value,
-            TypeCodes.ULC_COMP.value,
-            TypeCodes.CCC_COMP.value
-        ],
-        LearBusinessTypes.EXTRA_PRO_A.value: [
-            TypeCodes.EXTRA_PRO_A.value
-        ],
+    NUMBERED_CORP_NAME_SUFFIX = {
+        TypeCodes.BCOMP: 'B.C. LTD.',
+        TypeCodes.BC_COMP: 'B.C. LTD.',
+        TypeCodes.ULC_COMP: 'B.C. UNLIMITED LIABILITY COMPANY',
+        TypeCodes.CCC_COMP: 'B.C. COMMUNITY CONTRIBUTION COMPANY LTD.',
```

Comment:
For what it's worth, the UI doesn't add "LTD" to the name.

In shared repo: https://github.com/bcgov/bcrs-shared-components/blob/83cfe9c8dbe19714567814eb3efd7e08f35faf49/src/components/correct-name/CorrectAmlNumbered.vue#L44

In Create UI: https://github.com/bcgov/business-create-ui/blob/1dee380d0e31eaa53a9fb4392f844ab769f147df/src/components/common/NameRequestInfo.vue#L282

In MBR:
![image](https://github.com/bcgov/lear/assets/10002889/c7fcf3be-6ac3-481f-a40c-45325196f864)

Reply:
This is what we implemented in legal-api. We can check with the business

Reply:
Yes, please ask Naveen.

Reply:
I found this in colin
![image](https://github.com/bcgov/lear/assets/64869009/f8eadb14-36d4-4a98-9c93-708fadee4ce5)

Reply:
@severinbeauvais  Will go with LTD. 
![image](https://github.com/bcgov/lear/assets/64869009/72c4949b-1e33-4afd-8756-563e19d703d6)

Reply:
I've created ticket https://github.com/bcgov/entity/issues/21894 to fix the UI accordingly.

THIS conversation is resolved.